changelog/61860.fixed

```diff
@@ -0,0 +1 @@
+Fixed KeyError when trying to set some policies
```

Comment:
Can you elaborate for what module this impacts.

Reply:
Done